compiler/src/main/scala/dx/translator/Extras.scala

```diff
@@ -461,15 +461,24 @@ object Extras {
   private val RuntimeAttrs =
     Set(
         "dx_instance_type",
+        // WDL default runtime attributes
         "memory",
         "disks",
         "cpu",
         "docker",
         "container",
-        // TODO: I think these are no longer allowed - they have been moved to top-level
```

Comment:
Not sure what's the "top-level", but I trust your judgement here.

Reply:
Thanks! The top-level means they are top-levels attrs of the Extras class now.
https://github.com/dnanexus/dxCompiler/blob/e9fc37c5c1e82da9dc971d42de71be0347c9d11f/compiler/src/main/scala/dx/translator/Extras.scala#L429-L437